e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+import { TabsPage } from '../pages/tabs-page';
+import { ButtonsPage } from '../pages/buttons-page';
+
+test.describe('AngularJS-Webpack-Babel app', () => {
+  // eslint-disable-next-line
+  test.beforeEach(async ({ page }) => {
+    await page.goto('http://localhost:8083/');
```

Comment:
I think it would be better to use relative paths to navigate to pages in tests:

```suggestion
    await page.goto('');
```

Reply:
Changed